pypi-setuptools.yaml

```
@@ -0,0 +1,60 @@
+package:
+  name: pypi-setuptools
+  version: 69.2.0
+  epoch: 1
+  description: Easily download, build, install, upgrade, and uninstall Python packages
+  copyright:
+    - license: "MIT"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - python-3.10-base
+      - python-3.11-base
+      - python-3.12-base
+      - wolfi-base
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+vars:
+  pypi-package: setuptools
+
+data:
+  - name: py-versions
+    items:
+      3.12: ""
+      3.11: ""
+      3.10: ""
```

```suggestion
      3.12:
      3.11:
      3.10:
```
If you don't use the value, you can omitted.